javascript/big-practice/src/scripts/views/listNoteView.js

```diff
@@ -71,29 +134,52 @@ export default class ListNoteView {
    * @param {function} handler is fis a function with 2 values are
    *  title and description
    */
-  handleClickOut = (e, handler) => {
-    const note = document.querySelector('.form-input');
+  handleClickOut(e, handler) {
+    const note = selectDOMClass('.form-input');
     if (e.target.contains(note)) {
       this.addNewNote(handler);
+      this.inputAddElement.style.height = '1px';
+      this.inputTitleElement.style.height = '1px';
     }
-  };
+  }
 
   /**
    * @description function show input form
    */
-  showInputForm = () => {
+  showInputForm() {
     this.inputAddElement.addEventListener('focus', () => {
       this.formUtilitiesElement.classList.remove('hide');
       this.formTitleElement.classList.remove('hide');
     });
-  };
+  }
+
+  /**
+   * @description function add element note and hide input after adding
+   *
+   * @param {function} handler is a function with 2 values are
+   *  title and description
+   */
+  addNewNote(handler) {
+    const title = selectDOMClass('.note-title').value;
+    const description = selectDOMClass('.note-description').value;
+
+    if (!title && !description) {
```

Comment:
The logic code when you add class `hide` to DOM is redundant, please help me check